docs/dev/howtos/locally-published-components-in-fenix.md

```diff
@@ -47,10 +47,20 @@ git clone https://github.com/mozilla-mobile/fenix/
     3. Inside [`build.gradle`][android-components-build-gradle], add
        `mavenLocal()` inside `allprojects { repositories { <here> } }`.
 
-    4. Inside the android-component's `local.properties` file, ensure
+    4. Add the following block in the [`settings.gradle`](https://github.com/mozilla-mobile/android-components/blob/d67f83af679a2e847e5bd284ea4a30b412403241/settings.gradle#L7) file, in order to have the Glean Gradle plugin loaded from the local Maven repository.
```

Comment:
nit: I have tried following these directions and one confusing part was where exactly to put this code block. I added it to end of the file and got and error saying it should come before anything else. Maybe change this text to:
> Add the following block [...] before any other statement in the script, in order to [...]